tests/unit/backend/morph/test_crop.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 import numpy as np
 import pytest
-from imars3d.backend.morph.crop import crop, detect_bounds
+from imars3d.backend.morph.crop import crop
```

Comment:
if you can move `detect_bounds` back to the module level, some of the changes here will no longer be necessary.